src/Resources/Resources/help/New-AzManagementGroupDeploymentStack.md

```diff
@@ -187,7 +187,7 @@ Action to take on resources that become unmanaged on deletion or update of the d
 Type: Microsoft.Azure.Commands.ResourceManager.Cmdlets.SdkModels.DeploymentStacks.PSActionOnUnmanage
 Parameter Sets: (All)
 Aliases:
-Accepted values: DetachAll, DeleteResources, DeleteResourcesAndResourceGroups, DeleteAll
+Accepted values: DetachAll, DeleteResources, DeleteAll
```

Comment:
Will commands be broken by this deletion?

Reply:
This PR will break the customers who are using `DeleteResourcesAndResourceGroups`, do you have any reason to remove this value?